### PR TITLE
better defaults

### PR DESCRIPTION
**Pull request by ziofil**
Now `settings.HBAR` defaults to 1.0 and `settings.AUTOSHAPE_PROBABILITY` defaults to 0.99999 (99.999%).

### CODE REVIEW

File: mrmustard/lab_dev/transformations/amplifier.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ class Amplifier(Channel):
         >>> from mrmustard.lab_dev import Amplifier, Coherent
 
         >>> amp = Amplifier([0], gain=4)
-        >>> coh = Coherent([0], x=1.0, y=2.0)  # units of hbar=2 (default)
+        >>> coh = Coherent([0], x=1.0 * np.sqrt(2), y=2.0 * np.sqrt(2))  # units of hbar=1 (default)
```
Comment (apchytr) [inline mrmustard/lab_dev/transformations/amplifier.py:44]: Thinking about user convenience should we try to make x and y hbar independent?

File: mrmustard/utils/settings.py [hunk 2]
```diff
@@ -15,7 +15,7 @@
 """A module containing global settings."""
 
 from __future__ import annotations
-
+from typing import Optional
```
Comment (apchytr) [inline mrmustard/utils/settings.py:18]:
```suggestion
```

File: mrmustard/utils/settings.py [hunk 3]
```diff
@@ -153,15 +149,15 @@ def HBAR(self, value: float):
...
+    def SEED(self) -> int:
         r"""Returns the seed value if set, otherwise returns a random seed."""
         if self._seed is None:
             self._seed = np.random.randint(0, 2**31 - 1)
             self.rng = np.random.default_rng(self._seed)
         return self._seed
 
     @SEED.setter
-    def SEED(self, value: int):
+    def SEED(self, value: Optional[int]):
```
Comment (apchytr) [inline mrmustard/utils/settings.py:160]:
```suggestion
    def SEED(self, value: int | None):
```

File: mrmustard/lab_dev/transformations/amplifier.py (hunk 1 above)
Comment (ziofil) [reply mrmustard/lab_dev/transformations/amplifier.py:44]: The Bargmann representation is hbar-independent but coordinates in phase space are hbar-dependent by definition. I either had to put a sqrt(2) here or change the allclose statement below where I check the value of mu (which is hbar dependent). I'll change the value below to make it clear.

File: mrmustard/lab_dev/transformations/amplifier.py [hunk 4]
```diff
@@ -41,7 +41,7 @@ class Amplifier(Channel):
...
         >>> _, mu, _ = (coh >> amp).phase_space(0)
         >>> assert np.allclose(mu[0], np.array([4.0, 8.0]))
```
Comment (ziofil) [inline mrmustard/lab_dev/transformations/amplifier.py:46]:
```suggestion
        >>> coh = Coherent([0], x=1.0, y=2.0)
        >>> _, mu, _ = (coh >> amp).phase_space(0)
        >>> assert np.allclose(mu[0]*np.sqrt(2/settings.HBAR), np.array([4.0, 8.0]))
```

**Review by apchytr — APPROVED** [review]

**Pull request merged**